clientd3d/music.c

```diff
@@ -132,9 +136,11 @@ void MusicClose(void)
 //	Not used by MSS version
 /*
  * OpenMidiFile:  Open midi file for playing.
+ *   For background room music opens midi_bg_music_element device.
+ *   For any other music (jala) opens midi_element device.
  *   Returns 0 if successful; MCI error code otherwise.
  */
-DWORD OpenMidiFile(const char *lpszMIDIFileName)
+DWORD OpenMidiFile(const char *lpszMIDIFileName, UINT device)
```

Comment:
I'm confused what the new "device" parameter means in this function.  The new documentation talks about "midi_bg_music_element" and "midi_element" devices.  Is "device" the same as one of these?  Does it mean something else?

Reply:
I've added a comment on why there is a device argument.  OpenMidiFile is used for background music and for gameplay action related music.  The argument tells us what to name the device when we open a new midi file so that we can always juggle the two MCI devices correctly regardless of any convoluted user actions.